doc/articles/platform-specific-xaml.md

```diff
@@ -129,30 +129,24 @@ The pre-defined prefixes are listed below:
 
 More visually, platform support for the pre-defined prefixes is shown in the below table:
 
-| Prefix        |  Win  | Droid |  iOS  |  Web  | macOS | Skia  |
+| Prefix        |  Windows  | Android |  iOS  |  Web  | macOS | Skia  |
 |---------------|-------|-------|-------|-------|-------|-------|
 | `win`         | ✔ | ✖ | ✖ | ✖ | ✖ | ✖ |
 | `android`     | ✖ | ✔ | ✖ | ✖ | ✖ | ✖ |
 | `ios`         | ✖ | ✖ | ✔ | ✖ | ✖ | ✖ |
 | `wasm`        | ✖ | ✖ | ✖ | ✔ | ✖ | ✖ |
 | `macos`       | ✖ | ✖ | ✖ | ✖ | ✔ | ✖ |
 | `skia`        | ✖ | ✖ | ✖ | ✖ | ✖ | ✔ |
-| `xamarin`     | ✖ | ✔ | ✔ | ✔ | ✔ | ✔ |
```

Comment:
Note: I'm aware `xamarin` is supported, but it's the same as `not_win` and I'm intentionally removing it as we should be encouraging people to use `not_win`. We may even deprecate `xamarin` prefix in the future as its name doesn't really reflect the reality.